oprf/oprf_test.go

```diff
@@ -0,0 +1,168 @@
+package oprf
```

Comment:
```suggestion
package oprf_test
```

Reply:
Mmm.. but then the oprf functions will not be available to this package..

Reply:
You need to import the package, as any other user will do

Reply:
Done.

Reply:
Making it part of another package makes it difficult to test functions or parameters that are not exported. So, I'll prefer to keep it part of the same package.